### PR TITLE
Adds toe stubbing

### PR DESCRIPTION
**Pull request by ike709**
## Changelog
:cl:
add: There is a small chance that you will stub your toe if you run into a table without shoes.
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
Based

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -34,6 +34,15 @@
 	smooth = SMOOTH_TRUE
 	canSmoothWith = list(/obj/structure/table, /obj/structure/table/reinforced)
 
+/obj/structure/table/Bumped(mob/living/carbon/human/H)
+	. = ..()
+	if(!istype(H) || H.shoes)
```
Comment (zeskorion) [inline code/game/objects/structures/tables_racks.dm]: maybe check if the target has a foot, or maybe just if they're crawling

File: code/game/objects/structures/tables_racks.dm [hunk 2]
```diff
@@ -34,6 +34,15 @@
...
 	canSmoothWith = list(/obj/structure/table, /obj/structure/table/reinforced)
 
+/obj/structure/table/Bumped(mob/living/carbon/human/H)
+	. = ..()
+	if(!istype(H) || H.shoes)
+		return ..()
+	if(prob(5))
+		to_chat(H, "<span class='warning'>You stub your toe on the [name]!</span>")
+		H.apply_damage(2, BRUTE, def_zone = pick(BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+		H.Paralyze(20)
```
Comment (zeskorion) [inline code/game/objects/structures/tables_racks.dm:45]: H.emote("scream")?

**Review by zeskorion — DISMISSED** [review]

**Comment by Pill-U** [comment]
random tripping when?

**Review by zeskorion — APPROVED** [review]

**Comment by Cenrus** [comment]
Change it so that you stub your toe on communications consoles if you call the shuttle really early into the shift.

**Comment by PowerfulBacon** [comment]
How does this make any sense, how do you stub your toe on a table?
(Also we need backpacks getting caught in door handles)

**Comment by ike709** [comment]
Have you never stubbed your toe on a table or desk leg?

**Comment by PowerfulBacon** [comment]
> Have you never stubbed your toe on a table or desk leg?

No, never for me it's always running into doors.

**Comment by ike709** [comment]
You haven't lived.

**Comment by PowerfulBacon** [comment]
Alos, if you keep walking into it, like If your keyboard gets stuck, will it just fuck you up?

**Comment by ike709** [comment]
I don't think that's how `Bumped()` works (it's 3am, I may be wrong) but it is only a `prob(5)` chance regardless and does negligible damage.

**Review by zeskorion — APPROVED** [review]

**Comment by theXDkidoflol** [comment]
bro this is going to RUIN BEESTATION

**Comment by zeskorion** [comment]
> bro this is going to RUIN BEESTATION

well, mister theXDkidoflol, no

**Pull request merged**